test/Tmds.Ssh.Tests/SshServer.cs

```diff
@@ -20,6 +20,7 @@ public class SshServer : IDisposable
     public string TestUserHome => $"/home/{TestUser}";
     public string TestUserPassword => "secret";
     public string TestUserIdentityFile => $"{ContainerBuildContext}/user_key_rsa";
+    public string TestSubsystem = "tmds_test";
```

Comment:
nit: rename `tmds_test` to `test_subsystem`

Reply:
Has been renamed.